plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopHang.java

```diff
@@ -39,7 +38,7 @@
     public void setUp()
             throws Exception
     {
-        File tempPath = createTempDir();
+        File tempPath = Files.createTempDirectory("tmp").toFile();
```

Comment:
Do we need to call `tempPath.deleteOnExit()`? Does this file remain after JVM exits?

Reply:
Good catch. We should do 
```java
deleteRecursively(tempPath.toPath(), ALLOW_INSECURE);
```
in `tearDown()` like we do in other places, such as `ElasticsearchServer`.

Note that `File.deleteOnExit()` is an anti-pattern that we should avoid (and probably ban in modernizer).

Making sure we delete the directories is not strictly part of removing the deprecated function, but since we're auditing all of these anyway, it would be nice to fix now as well, if @hguercan doesn't mind.

Reply:
Should it be applied for all tests? If yes: If the temp folder is created in the test method itself and the test class does not have a tearDown/afterClass, how should we proceed with this temp folders?

Reply:
checked other classes: they are deleting the files, if not I added deletion